### DISCUSSION TITLE
Change background color of floating windows

### DISCUSSION THREAD

**Discussion by RST-J** [neoclide/coc.nvim#5085] [Q&A] [answered]
Hello and apologies for if that is the wrong place to come to with my question. 

I'm just too stupid to understand how the `highlight` option for `floatFactory.floatConfig' works.

My problem is that floating windows have the exact same background color as my vim/terminal background which makes them indistinguishable. I wanted to change the background color but nothing else from the default highlighting and couldn't get that working. Can someone tell me what I have to put as value to `highlight` so that everything stays as it is except that `ctermbg=14` is used? I'd also be glad to define a custom highlight group for that that is derived from `CocFloating` and assign that but I absolutely know nothing about how to do that and couldn't find out. :/

**Comment by fannheyward** [comment, marked as answer]
The highlight group of floating window is `CocFloating`, change this and try again.

**Reply by RST-J** [reply]
@fannheyward how can I change it? I'd redefine it if I knew what else is in its definition but I don't know how to find out.

**Reply by RST-J** [reply]
I could find the spot where it was last modified using `:verbose hi CocFloating` and went from there.
